app/assets/javascripts/admin/tagify.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
@garytaylor @leoapost I'm ok with pure JS. But if we are going to convert to pure js from coffee script it should be in a new story :) Happy to discuss this further.